package.json

```diff
@@ -21,6 +21,7 @@
     "react-router-dom": "4.3.1",
     "react-router-hash-link": "1.2.1",
     "react-scripts": "2.0.5",
+    "react-text-loop": "^1.1.0",
```

Comment:
Let's peg versions to exact going forward and start using Greenkeeper to handle dependency management. https://greenkeeper.io